Marlin/Conditionals_post.h

```diff
@@ -824,4 +824,12 @@
     #define FMOD(x, y) fmodf(x, y)
   #endif
 
+  #if defined(TEENSYDUINO)
```

Comment:
Teensy core changes the min/max implementation in a way that we cannot use it. This changes it to the way AVR core does it.